modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -330,7 +330,12 @@ const setup = (editor: Editor): RenderInfo => {
 
   const uiMothership = Gui.takeover(sink);
 
-  Events.setup(editor, mothership, uiMothership);
+  const resizeMothership = () => {
```

Comment:
[nitpick] Considering line 338 has both `mothership` and `uiMothership` and we're using the latter, maybe this should be called `resizeUiMothership` for readability?